addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/discovery/LGWebOSDiscovery.java

```diff
@@ -135,7 +136,11 @@ public void onDiscoveryFailed(DiscoveryManager manager, ServiceCommandError erro
     // Helpers for DiscoveryManagerListener Impl
     private DiscoveryResult createDiscoveryResult(ConnectableDevice device) {
         ThingUID thingUID = createThingUID(device);
-        return DiscoveryResultBuilder.create(thingUID).withLabel(device.getFriendlyName()).build();
+        Map<String, Object> properties = new HashMap<>(1);
+        properties.put(PROPERTY_DEVICE_ID, device.getId());
+
+        return DiscoveryResultBuilder.create(thingUID).withLabel(device.getFriendlyName()).withProperties(properties)
```

Comment:
You could remove a few lines of code when using: `.withProperty(PROPERTY_DEVICE_ID, device.getId())`

Reply:
yes, already have this in my code locally.